WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -254,11 +254,6 @@ class NotificationsViewController: UIViewController, UIViewControllerRestoration
 //            self.showInlinePrompt()
 //        }
 
-        if !unreadNotificationIds.isEmpty {
-            // Track as significant event for App Rating calculations
-            AppRatingUtility.shared.incrementSignificantEvent()
-        }
```

Comment:
This code was supposed to increment the significant events count when the user visits the Notifications screen and has unread notifications. But that didn't work, the `unreadNotificationIds` is empty when I visit the screen even though there are unread notifications screen. 

After some digging, I realized the `unreadNotificationIds` is populated when the user switches to `Unread` tab, then it is emptied when switching to another tab. 

This behavior is buggy in my opinion, and I replaced with the "User taps on an unread notification" action.

cc @hassaanelgarem